flink/src/main/java/org/apache/iceberg/flink/IcebergTableSource.java

```diff
@@ -102,6 +107,21 @@ public String explainSource() {
     if (projectedFields != null) {
       explain += ", ProjectedFields: " + Arrays.toString(projectedFields);
     }
+
+    if (isLimitPushDown) {
+      explain += String.format(", LimitPushDown %s, Limit %d", isLimitPushDown, limit);
```

Comment:
Here the `isLimitPushDown` is always `true`, how about use the string `String.format(", LimitPushDown: %d", limit)` ?